test/test_proxy_lib.cpp

```diff
@@ -31,5 +31,5 @@ TEST_F(test, proxyLibBasic) {
 #else
     size_t size = ::malloc_usable_size(ptr);
 #endif
-    UT_ASSERTeq(size, 0xDEADBEEF);
+    ASSERT_EQ(size, 0xDEADBEEF);
```

Comment:
I guess you can do more changes then just one file at the time :)

Reply:
I have added more files to this PR